scalableminds/webknossos
# Floor the z coordinate on initial movement-shortcut-firing

### URL of deployed dev instance (used for testing):
- https://floorzwhenfing.webknossos.xyz

### Steps to test:
- Open a dataset and enforce a z position which is very close to it's higher neighbor (e.g., `9.9`) via
```
webknossos.apiReady().then(async api => {
	api.tracing.centerPositionAnimated([28928, 28928, 1282.9], false)
})
```
- press (and keep pressing f) and notice how the z value changes in a consistent manner (on current prod, the first z changes are faster in comparison)

### Issues:
- fixes https://discuss.webknossos.org/t/move-value-not-consistent/1645/2

------
